arcgis-ios-sdk-samples/Layers/Web tiled layer/README.md

```diff
@@ -1,12 +1,40 @@
 # Web tiled layer
 
-This sample demonstrates how to display map tiles from an online resource
+Display a tiled web layer.
 
-![](image1.png)
+![Web tiled layer sample](web-tiled-layer.png)
+
+## Use case
+
+Tiled map services are a set of pre-generated images (e.g. "tiles") arranged in folders for each row, column, and zoom level. As you navigate the map, map tiles are requested for the current extent. `AGSArcGISTiledLayer` and `AGSWMTSLayer` are types of tiled map services used for specific data types. `AGSWebTiledLayer` is useful for displaying other data sources that contain tiles arranged in a row/column/level directory structure, such as OpenStreetMap.
+
+## How to use the sample
+
+Run the sample and a map will appear. As you navigate the map, map tiles will be fetched automatically and displayed on the map.
 
 ## How it works
 
-`AGSWebTiledLayer` provides a simple way to integrate non-ArcGIS Services as a layer in a map. In this case, map tiles from Stamen are added to the map. The template URL is specified by setting the subDomains, level, col, and row attributes. Additionally, copyright information is added to the layer so that the layer can be properly attributed. The layer is added to an `AGSBasemap`, and the basemap is added to an `AGSMap`. Finally, the map is set on the `AGSMapView`, and the tiled layer is displayed.
+Web tiled services use a uniform addressing scheme with pre-rendered tiles. Image tiles are accessed via a URL template string, with parameters for subdomain, level, column, and row.
+* Subdomain is optional and allows Runtime to balance requests among multiple servers for enhanced performance.
+* Level, row, and column select the tiles to load based on the visible extent of the map.
+
+1. Create an `AGSWebTiledLayer` from a URL and a list of subdomains.
+2. Create an `AGSBasemap` from the layer.
+3. Update the attribution on the layer. Note: this is a necessary step because web tiled services don't have associated service metadata.
+
+For more information about web tiled layers, see the following resources:
+* [Wikipedia: tiled web maps](https://en.wikipedia.org/wiki/Tiled_web_map)
+* [ArcGIS Pro: Share a web tile layer](http://pro.arcgis.com/en/pro-app/help/sharing/overview/web-tile-layer.htm)
```

Comment:
```suggestion
* [ArcGIS Pro: Share a web tile layer](https://pro.arcgis.com/en/pro-app/help/sharing/overview/web-tile-layer.htm)
```
See if it can be https.